rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -56,7 +56,10 @@ extern "C" RocalTensor ROCAL_API_CALL rocalJpegFileSource(RocalContext context,
                                                           bool shuffle = false,
                                                           bool loop = false,
                                                           RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MOST_FREQUENT_SIZE,
-                                                          unsigned max_width = 0, unsigned max_height = 0, RocalDecoderType rocal_decoder_type = RocalDecoderType::ROCAL_DECODER_TJPEG, std::pair<RocalLastBatchPolicy, bool> last_batch_info = {RocalLastBatchPolicy::ROCAL_LAST_BATCH_FILL, true});
+                                                          unsigned max_width = 0, unsigned max_height = 0, RocalDecoderType rocal_decoder_type = RocalDecoderType::ROCAL_DECODER_TJPEG,
+                                                          bool stick_to_shard = false,
```

Comment:
I suggest passing a struct instead of individual variables for last_batch_policy. This way if we change the parameters in future, API won't change.

Reply:
rocalJpegFileSource loader is usually used for full data-set. Does all sharding parameters matter here

Reply:
stick_to_shart = true by default is most usecases